BreakSentence.cs

```diff
@@ -26,8 +26,26 @@ public class DetectedLanguage
 
     class Program
     {
+        private const string key_var = "TRANSLATOR_TEXT_SUBSCRIPTION_KEY";
+        private static readonly string subscription_key = Environment.GetEnvironmentVariable(key_var);
```

Comment:
should this be subscriptionKey instead of subscription_key?

Reply:
Done for all samples.